library/ssl_tls.c

```diff
@@ -6407,9 +6407,11 @@ static int ssl_session_load( mbedtls_ssl_session *session,
     const unsigned char *p = buf;
     const unsigned char * const end = buf + len;
     int minor_ver = 0;
-#if defined(MBEDTLS_HAVE_TIME) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+#if defined(MBEDTLS_HAVE_TIME)  \
+    && defined(MBEDTLS_SSL_PROTO_TLS1_2_OR_EARLIER)
```

Comment:
Why was `MBEDTLS_SSL_SESSION_TICKETS` removed here?

Reply:
`start` is only used with `defined(MBEDTLS_HAVE_TIME)   && defined(MBEDTLS_SSL_PROTO_TLS1_2_OR_EARLIER)`.

I have removed `start` . I think that is better fix. That make code easy to maintain.